libibumad/libibumad.map

```diff
@@ -39,3 +39,9 @@ IBUMAD_1.0 {
 		umad_attribute_str;
 	local: *;
 };
+
+IBUMAD_1.1 {
+	global:
+		umad_get_ca_namelist;
+		umad_free_ca_namelist;
```

Comment:
Keep sorted
Also need to update the debian/libibumad3.symbols

Reply:
According to keep sorted:
Do you mean that the functions should be sorted
by Alphabetical order? ("umad_f*" before "umad_g*")?

Reply:
yes